doc/apache2/vhost.template

```diff
@@ -188,4 +188,9 @@
             #ExpiresDefault "now plus 30 days"
         #</LocationMatch>
     </IfModule>
+
+    # Enable gzip encoding
```

Comment:
apache default config for when mod_deflate already includes a few of these. Would it not be better to only add here the missing ones? Or at least add a comment mentioning mention that